.ci/pitest.sh

```diff
@@ -38,17 +38,25 @@ pitest-annotation|pitest-design \
 |pitest-packagenamesloader \
 |pitest-common-2|pitest-misc|pitest-xpath \
 |pitest-filters \
-|pitest-main \
 |pitest-coding \
 |pitest-regexp \
 |pitest-meta)
-  mvn -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
+  mvn --no-transfer-progress -e -P$1 clean test org.pitest:pitest-maven:mutationCoverage;
```

Comment:
please move to separate commit with prefix "minor: " (there are bunch of other commits like this)
we need to clearly separate changes for 101888 and other changes.

Reply:
Done